code/modules/mob/living/carbon/xenomorph/Xenomorph.dm

```diff
@@ -175,9 +125,8 @@
 				to_chat(user, "It is heavily injured and limping badly.")
 
 	if(hivenumber != XENO_HIVE_NORMAL)
-		if(hivenumber && hivenumber <= hive_datum.len)
-			var/datum/hive_status/hive = hive_datum[hivenumber]
-			to_chat(user, "It appears to belong to the [hive.prefix]hive")
+		var/datum/hive_status/hive = GLOB.hive_datums[hivenumber]
+		to_chat(user, "It appears to belong to the [hive.prefix]hive")
```

Comment:
```suggestion
		to_chat(user, "It appears to belong to the [hive.prefix] hive")
```

Reply:
prefix contains the space `	prefix = "Corrupted "`

Reply:
No biggie, but it would be cleaner for the space to be in the message.

Reply:
Then normal hives would get double space

Reply:
Then a negative space for them!
Ok, fair enough.